### PR TITLE
Show course & unit titles and add pre-loader

### PR DESCRIPTION
**Pull request by tonymontaro**
#### What does this PR do?
This PR replaces the breadcrumbs on the course page with the course & unit titles. It also adds a pre-loader to the explorer page.

#### Description of tasks performed?
-  The course and unit titles are shown in the title section of the course page
-  Unit titles are hidden inside the course card
-  Added a pre-loader that is shown to the user while courses are being loaded in the explorer page
-  Fixed the extra spaces below carousel items
-  Darkened the hero banner image to make the text stand out
-  Updated the hero/blue section text on the homepage
-  Optimized new images for web (e.g the hero image size changed from 1.1mb -> 400kb)
-  Updated the blue section text to look better
-  Ensured that how-it-works images have the same width and height

#### How to test ?
Visit a course page (e.g: https://codeforafrica-coursebuilder.appspot.com/audience_engagement/unit?unit=1&lesson=3)

#### Screenshot ?
cfa-mooc-topics

Pre-loader:
cfa-mooc-pre-loader

### CODE REVIEW

File: coursebuilder/modules/explorer/_static/components/course-explorer/course-explorer.html [hunk 1]
```diff
@@ -1,9 +1,10 @@
 <link rel="import" href="/static/polymer-1.2.0/polymer/polymer.html">
-<link rel="import" href="/static/polymer-1.2.0/iron-ajax/iron-ajax.html" />
-<link rel="import" href="/static/polymer-1.2.0/paper-spinner/paper-spinner.html" />
+<link rel="import" href="/static/polymer-1.2.0/iron-ajax/iron-ajax.html"/>
+<link rel="import" href="/static/polymer-1.2.0/paper-spinner/paper-spinner.html"/>
 <link rel="import" href="/modules/explorer/_static/components/utility/unsafe-html.html">
 <link rel="import" href="/modules/explorer/_static/components/utility/floating-sidebar.html">
-<link rel="import" href="/modules/explorer/_static/components/course-cards/course-card-grouper.html">
+<link rel="import"
```
Comment (chegejames) [inline coursebuilder/modules/explorer/_static/components/course-explorer/course-explorer.html]: Indentation

**Review by chegejames — CHANGES_REQUESTED** [review]
Please add screenshots to the description.

**Review by chegejames — APPROVED** [review]

**Review by chegejames — CHANGES_REQUESTED** [review]
@andela-angene Intruders?

coursebuilder/modules/explorer/_static/images/course/course2.png

coursebuilder/modules/explorer/_static/images/course/course3.jpg

**Review by chegejames — APPROVED** [review]

File: coursebuilder/modules/explorer/_static/components/course-explorer/course-explorer.html (hunk 1 above)
Comment (tonymontaro) [reply coursebuilder/modules/explorer/_static/components/course-explorer/course-explorer.html]: Updated.

**Pull request merged**